source/manual/github-unavailable.html.md

```diff
@@ -83,17 +36,3 @@ To fix this:
 1. If you are prompted to add the item to keychain, deny.
 
 There is more information about setting up your access key in the [AWS guide](https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up-https-unixes.html)
-
-## Drill creating and deploying a branch from CodeCommit
```

Comment:
This is linked to from https://docs.publishing.service.gov.uk/manual/2nd-line-drills.html#deploy-from-aws-codecommit-when-github-is-unavailable

By deleting this section, are you saying it's no longer possible to drill deploying from CodeCommit?

Indeed, is it no longer possible to deploy anything when GitHub is down? ISTR an offline chat with Sean suggesting it could still be done but would require building images locally and pushing the images up to the store. Can we document that process instead?

Reply:
The tools for deploying from codecommit (git and jenkins) are being decommissioned so yes that drill is no longer possible. 
I'll speak with @theseanything and confirm if there is a way to deploy when github is down.

Reply:
I've updated the process with a way to deploy previously deployed versions of code that are already stored in ECR.